src/azure-cli/azure/cli/command_modules/batch/_params.py

```diff
@@ -382,7 +382,7 @@ def load_arguments(self, _):
 
     with self.argument_context('batch pool create') as c:
         c.argument('json_file', help='The file containing pool create properties parameter specification in JSON(formatted to match REST API request body). If this parameter is specified, all \'Pool Create Properties Parameter Arguments\' are ignored.  See https://learn.microsoft.com/rest/api/batchservice/pool/add?tabs=HTTP#request-body')
-        c.argument('resource_tags', arg_group='Pool', type=resource_tag_format, help="User is able to specify resource tags for the pool. Any resource created for the pool will then also be tagged by the same resource tags")
+        c.argument('resource_tags', action=None, arg_group='Pool', type=resource_tag_format, help="User is able to specify resource tags for the pool. Any resource created for the pool will then also be tagged by the same resource tags")
```

Comment:
This would be a good place to put a comment about why `action=None` is there. Something like:

`Kludge: Add action=None to work around an error registering a breaking change for this argument`

Reply:
Added comment